entity/scenario_entities/src/entitiy_manager.cpp

```diff
@@ -90,10 +92,11 @@ try
   }
   else
   {
-    auto plugin = loader.createInstance(*iter);
+    auto plugin = loader.createSharedInstance(*iter);
     plugin->configure(node, api_ptr);
-    entities_.push_back(plugin);
+    entities_.emplace_back(plugin);
   }
+  return true;
```

Comment:
Reviewer: please let me know what should be returned here and feel free to edit.

Reply:
@mitsudome-r I will report correct return value by today.

Reply:
I think returning true is fine.
Maybe also adding `return false` at the end of `catch (...)` statement makes more sense.

Reply:
I think we'd get an unreachable statement error with that, since the macro rethrows the error.